src/components/unique-token/UniswapV3NFT.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useRef } from 'react';
+import WebView from 'react-native-webview';
+import { useTheme } from '../../context/ThemeContext';
+import { magicMemo } from '../../utils';
+import Spinner from '../Spinner';
+import { Centered } from '../layout';
+import { position } from '@rainbow-me/styles';
+
+const UniqueTokenImage = ({ url }) => {
+  const { colors } = useTheme();
+  const [ready, setReady] = useState(false);
+
+  const jsToExecute = `
+    document.getElementsByTagName('svg')[0].setAttribute('height', '100%');
+    document.getElementsByTagName('svg')[0].setAttribute('width', '100%');    
+    true;
+   `;
+
+  const webRef = useRef(null);
+
+  useEffect(() => {
```

Comment:
maybe you can make js logic a bit more reliable instead of this: 

E.g,


```
white(!document.getElementsByTagName('svg').length === 0) {
	// actively waiting
}
document.getElementsByTagName('svg')[0].setAttribute('height', '100%');
document.getElementsByTagName('svg')[0].setAttribute('width', '100%');    
window.postmessage('ready')
```

and `onMessage={() => setReady(true)}` on RN side